src/clj/athens/self_hosted/web/datascript.clj

```diff
@@ -32,8 +32,10 @@
     :datascript/left-sidebar-drop-above
     :datascript/left-sidebar-drop-below
     :datascript/unlinked-references-link
-    :datascript/unlinked-references-link-all
-    ;; TODO: all the events
+    :datascript/unlinked-references-link-all})
```

Comment:
is it automatic formatting that changes that?

Reply:
It's when I merged. Fixed.